applications/StructuralMechanicsApplication/custom_conditions/base_load_condition.cpp

```diff
@@ -321,6 +321,60 @@ namespace Kratos
         return IntegrationPoints[PointNumber].Weight() * detJ;
     }
 
+    //***********************************************************************************
+    //***********************************************************************************
+
+    void BaseLoadCondition::AddExplicitContribution(const VectorType& rRHS, 
+        const Variable<VectorType>& rRHSVariable,
+        Variable<array_1d<double,3> >& rDestinationVariable, 
+        const ProcessInfo& rCurrentProcessInfo)
+    {
+        KRATOS_TRY
+
+        const unsigned int number_of_nodes = GetGeometry().PointsNumber();
+        const unsigned int dimension       = GetGeometry().WorkingSpaceDimension();
+
+        if( rRHSVariable == EXTERNAL_FORCES_VECTOR && rDestinationVariable == EXTERNAL_FORCE )
```

Comment:
i am a bit lost about the meaning of this if....

Reply:
yeah I forgot to delete that, fixed